tabpfn_client/tabpfn_classifier.py

```diff
@@ -52,6 +52,8 @@ def init():
             # prompt for login / register
             PromptAgent.prompt_and_set_token(user_auth_handler)
 
+        PromptAgent.prompt_retrieved_messages(user_auth_handler.retrieve_messages())
```

Comment:
I suppose if there's no new messages, it wouldn't print anything right?
Maybe include a comment to clarify this.